source/features/add-timezone-to-hovercard.js

```diff
@@ -0,0 +1,103 @@
+import {h} from 'dom-chef';
+import select from 'select-dom';
+import moment from 'moment-timezone';
+import domify from '../libs/domify';
+
+function clockIcon() {
+	return (
+		<svg
+			class="octicon octicon-clock text-gray-light ml-5"
+			width="20"
+			height="16"
+			viewBox="0 0 14 16"
+			version="1.1"
+			aria-hidden="true"
+		>
+			<path
+				fill-rule="evenodd"
+				d="M8 8h3v2H7c-.55 0-1-.45-1-1V4h2v4zM7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7z"
+			/>
+		</svg>
+	);
+}
+
+function createTimezoneRequestURL(lat, lng) {
+	const timezoneAPI = 'https://maps.googleapis.com/maps/api/timezone/json?location=';
+	return `${timezoneAPI}${lat},${lng}&timestamp=1331161200&sensor=false`;
```

Comment:
I think the timestamp should be `Date.now() / 1000`

Reply:
Maybe it's just me, but the request seems to fail more often with this change 🤔

Reply:
Maybe it's just me, but this request seems to fail more often with `Date.now() / 1000` 🤔